codec-http2/src/main/java/io/netty/handler/codec/http2/DelegatingDecompressorFrameListener.java

```diff
@@ -175,6 +176,12 @@ protected EmbeddedChannel newContentDecompressor(final ChannelHandlerContext ctx
             return new EmbeddedChannel(ctx.channel().id(), ctx.channel().metadata().hasDisconnect(),
                     ctx.channel().config(), ZlibCodecFactory.newZlibDecoder(wrapper));
         }
+        if ("br".equalsIgnoreCase(contentEncoding.toString())) {
+            final ZlibWrapper wrapper = strict ? ZlibWrapper.ZLIB : ZlibWrapper.ZLIB_OR_NONE;
```

Comment:
How does the allocation of wrapper here work?

Reply:
Oops! My bad.